#! /bin/sh /usr/share/dpatch/dpatch-run
## 202_help.txt.diff; author unknown
##
## DP: * Because Perl is every time installed on Debian we can simplify the
## DP:   upgrade message.
## DP: * help.txt is installed as slrn-help.txt in Debian.

@DPATCH@
--- slrn-0.9.8.0.orig/src/slrn.c
+++ slrn-0.9.8.0/src/slrn.c
@@ -1221,9 +1221,8 @@
      {
 	slrn_message (_("\n! Your configuration file contains obsolete commands or function names that\n"
 		      "! will not be supported by future versions of this program.\n"
-		      "! If you have Perl installed, you can use the script slrnrc-conv to change\n"
-		      "! your configuration accordingly.  It can be found in the source distribution\n"
-		      "! or retrieved from <URL:http://slrn.sourceforge.net/data/>.\n"));
+		      "! You can use the script /usr/share/slrn/contrib/slrnrc-conv to change\n"
+		      "! your configuration accordingly.\n"));
 	Slrn_Saw_Warning = 1;
      }
 
@@ -1281,7 +1281,7 @@
 #ifdef VMS
 	slrn_snprintf (file, sizeof (file), "%s%s", SLRN_CONF_DIR, "help.txt");
 #else
-	slrn_snprintf (file, sizeof (file), "%s/%s", SLRN_CONF_DIR, "help.txt");
+	slrn_snprintf (file, sizeof (file), "%s/%s", SLRN_CONF_DIR, "slrn-help.txt");
 #endif
      }
 	
